src/alchemtest/amber/access.py

```diff
@@ -96,12 +94,60 @@ def load_invalidfiles():
         - 'data' : the example of invalid data files
         - 'DESCR': the full description of the dataset
 
+
+    .. deprecated:: 0.7
+       use :func:`load_testfiles` instead
+
     """
 
-    module_path = dirname(__file__)
-    data = [glob(join(module_path, 'invalidfiles/*.out.bz2'))]
+    warnings.warn(
+        "load_invalidfiles() was deprecated in 0.7.0 and will be removed in the following release."
+        " Use load_testfiles() instead",
+        DeprecationWarning)
+    module_path = Path(__file__).parent
+    data = [[
+        module_path / 'testfiles' / 'no_useful_data.out.tar.bz2',
```

Comment:
Just change these file names from `*.out.tar.bz2` to `*.out.bz2` and this PR is ready to be merged.

Reply:
Done!